src/components/monitor/monitor.jsx

```diff
@@ -36,15 +37,20 @@ const MonitorComponent = props => (
         holdToDisplay={props.mode === 'slider' ? -1 : 1000}
         id={`monitor-${props.label}`}
     >
-        <Draggable
-            bounds=".monitor-overlay" // Class for monitor container
+        <DraggableCore
             cancel=".no-drag" // Class used for slider input to prevent drag
-            defaultClassNameDragging={styles.dragging}
             disabled={!props.draggable}
+            onStart={props.onDragStart}
+            onDrag={props.onDrag}
             onStop={props.onDragEnd}
         >
             <Box
-                className={styles.monitorContainer}
+                className={classNames(styles.monitorContainer, {
+                    [styles.dragging]: props.dragging
+                })}
+                style={{
+                    transform: `translate(${props.x}px, ${props.y}px)`
```

Comment:
Will it cause any performance issues to re-render this component for every frame we're dragging?